app/views/katello/api/v2/products/_product_content.json.rabl

```diff
@@ -1,14 +1,5 @@
 attribute :enabled, :product_id
-node :content do |pc|
-  c = pc.content
-  {
-    id: c.cp_content_id,
-    name: c.name,
-    label: c.label,
-    vendor: c.vendor,
-    type: c.content_type,
-    gpgUrl: c.gpg_url,
-    contentUrl: c.content_url,
-    modifiedProductIds: c.modified_product_ids
-  }
+
+glue :content do |_content|
+  attributes :cp_content_id, :name, :label, :vendor, :content_type, :gpg_url, :content_url
```

Comment:
This is changing the output format of the API - losing the camel case and moving to the new instance methods. ID is also lost. I'm calling that out because we agreed on not breaking the API format when started caching the content in our db.

Reply:
Good catch, i didn't intend to change the camel casing (or removing the id)